LayoutTests/fast/editing/ruby-with-edited-text-crash.html

```diff
@@ -0,0 +1,27 @@
+<style>
+* { 
+    -webkit-user-modify: read-write;
+}
+</style>
+<script>
+    function main() {
+        if (window.testRunner)
+            testRunner.dumpAsText();
+        console.log('The test PASS if it does not crash.')
+        try { document.designMode = "on"; } catch { }
+        try { document.execCommand("selectAll", false, null); } catch { }
+        try { document.execCommand("italic", false, null); } catch { }
+        try { document.execCommand("italic", false, null); } catch { }
```

Comment:
could you omit these try/catch statements? I assume there's not to catch here.

Reply:
Removed